main.cpp

```diff
@@ -1,48 +1,36 @@
-//grade calculating algorithm for British universities
-//subject to covid-19 mitigations for UCL
-//covid-19 mitiations are: 6 year1 modules weighted at 1 out of 9, 4 year2 modules weighted at 3 out of 9, 8 year3 modules weighted at 5 out of 9.
+/*
+    grade calculating algorithm for British universities
+    subject to covid-19 mitigations for UCL
+    covid-19 mitiations are: 6 year1 modules weighted at 1 out of 9, 4 year2 modules weighted at 3 out of 9, 8 year3 modules weighted at 5 out of 9.
+*/
 
 #include <iostream>
+#include <iomanip>
```

Comment:
This header allows use of setprecision(int n), which rounds output double to n decimal places. 
Read [documentation](https://www.cplusplus.com/reference/iomanip/setprecision/)